src/main/java/frc/robot/constants/JoystickConstants.java

```diff
@@ -12,6 +12,7 @@ private JoystickConstants() {
 
     public static final int ROT_JOYSTICK_PORT = 0;
     public static final int TRANS_JOY_PORT = 2;
+    public static final int XBOX_PORT = 1; //TBD
```

Comment:
Can you expand on what's needed here?

Or a better approach would be to file an issue for the work that needs to be done, and then put a link to the issue in this comment.

Reply:
done: https://github.com/FRC-1294/frc2024/issues/241

Reply:
Add the link as a comment in the source code, instead of the TBD. Then you have full end-to-end cohesiveness and can track what needs to be done.

Reply:
resolved in new commit called "change xbox controller name..."